### PR TITLE
Improved Abductor/Abductee Objectives to be less of a meme.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

**Abductor:**
Now has an additional objective, an escape while alive objective, so they have to show their presence on the station.

**Abductees:**
Now get the following possible objective pairs:
Ensure that [50/50 ABDUCTOR OR OTHER ABDUCTEE] escapes alive and not in custody.
_OR_
Ensure that [NON-ABDUCTOR] is abducted and turned into an abductee.

If either objective fails to find targets, they are given a random objective found in ``strings/abductee_objectives.txt``

## Why It's Good For The Game
While the abductee objectives were good for LRP memes, such as objectives that tells you to "paint the station red with blood" or "communicate with machine noises", I do not think it's good for Skyrat as it seriously limits the freedom of players if they are given bad RNG and given an exceptionally shit objectives.

I have not been abducted yet so this isn't a salt PR. I've just been seeing some phenomenally shit objectives in the past week and feel that Abductors could be a little more on the RP side, so I gave both the abductors and abductees a reason to communicate/cooperate and be on the station more.

## Changelog
:cl: BurgerBB
tweak: Abductors now have an additional escape objective. Abductees now have non-meme objectives.
/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -1295,7 +1295,6 @@
 #include "code\modules\antagonists\_common\antag_team.dm"
 #include "code\modules\antagonists\abductor\abductor.dm"
 #include "code\modules\antagonists\abductor\abductee\abductee.dm"
-#include "code\modules\antagonists\abductor\abductee\abductee_objectives.dm"
```
Comment (Trilbyspaceclone) [inline tgstation.dme:1298]: ...

Comment (BurgerLUA) [reply tgstation.dme:1298]: ok boomer

**Comment by Useroth** [comment]
Not entirely sure if giving abductors escape objective is a good idea.

**Comment by BurgerLUA** [comment]
> Not entirely sure if giving abductors escape objective is a good idea.

It will get the abductors on the shuttle and with the crew. I know they're aliens and have a space ship, but part of their experients could be to blend in with society.

**Comment by ShadeAware** [comment]
Abductors are already an absolutely awful experience for everyone but the abductors themselves. The "_Make sure someone else is abducted_" objective is just... not good. 

Honestly, I think Abductors should be removed full stop, but thats for another PR.

**Comment by Akrilla** [comment]
Fix this up and we'll give this a try.

**Comment by BurgerLUA** [comment]
> Fix this up and we'll give this a try.

Other than merge conflicts, whats wrong with it?

**Comment by Akrilla** [comment]
Nothing, it's just the merge conflict...

**Comment by Akrilla** [comment]
travis issues

**Comment by BurgerLUA** [comment]
I forgot about this PR lmao.

**Pull request merged**